Extensions/Editor Extensions/Add Marketo Form/add_marketo_form.json

```diff
@@ -0,0 +1,82 @@
+{
+  "plugin_type": "widget",
+  "name": "Add Marketo Form",
+  "edit_page_url": "https://www.atticandbutton.us/",
+  "form_schema": [
+    {
+      "default_value": "https://app-123.marketo.com/js/forms2/js/forms2.min.js",
```

Comment:
Does the format of this url always have the same pattern? like "https://app-<NUMBER>.marketo.com/js/forms2/js/forms2.min.js"

Right now the extension code is kind of generically inserting a script from some location and if the user accidentally puts in the wrong URL, they could be loading the wrong or potentially even malicious script on their page.

One way around this might be to only allow users to configure the <NUMBER> so that we always guarantee that this extension adds a script from a marketo domain. However, this assumes marketo won't change the format of their URL.

Or we could rely on users to ensure they paste in the right script here as you are doing here.

I think this is fine either way, but would appreciate your thoughts.